packages/react-core/src/components/DataList/DataList.tsx

```diff
@@ -1,8 +1,19 @@
 import * as React from 'react';
 import { css } from '@patternfly/react-styles';
 import styles from '@patternfly/react-styles/css/components/DataList/data-list';
+import stylesGrid from '@patternfly/react-styles/css/components/DataList/data-list-grid';
+import { toCamel } from '../../helpers/util';
 import { PickOptional } from '../../helpers/typeUtils';
 
+export enum DataListGridBreakpoint {
+  gridNone = 'grid-none',
```

Comment:
I need to expose grid as well here 
https://github.com/patternfly/patternfly-react/issues/4965#issuecomment-704984236